### PR TITLE
[docs][cloud] Additional metrics

### PR DESCRIPTION
**Pull request by ddhodge**
Additional metrics added to Performance tab

@netlify /preview/yugabyte-cloud/cloud-monitor/overview/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 1]
```diff
@@ -61,30 +61,30 @@ The **Performance** tab provides the following metrics.
...
+| Disk IOPS/Sec | The number of disk input / output read and write operations per second averaged over each node. | Large spikes usually indicate large compactions. Rarely, in cases of a spiky workload, this could indicate block cache misses.<br>Because random reads always hit disk, you should increase IOPS capacity for this type of workload.<br>You should set an alert to a value much greater than your average or as a percentage of your available IOPS.<br>This value is averaged across all nodes in a cluster. An alert should be issued per node to detect source of underlying issues. |
+| Network Errors/Sec | The number of errors related to network packets received (RX) and transmitted (TX) per second, averaged over nodes. | If your environment produces a lot of errors, that could indicate an underlying infrastructure or operating system issue. |
+| Network Packets/Sec | The count of network packets received to the server (RX) and transmitted from the server (TX) per second, averaged over nodes. | Provides a view of the intensity of the network activity on the server. |
```
Comment (krishkar99) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:78]: Provides a view of the intensity of the network activity on the server. >> Represents a view of the amount of network traffic to and from the server.

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 2]
```diff
@@ -97,93 +97,59 @@ The **Performance** tab provides the following metrics.
...
+| YB-TServer Average Latency (ms) | Read: the average latency of read operations at the tablet level.<br>Write: the average latency of write operations at the tablet level. | When latency starts to degrade, performance may be impacted by the storage layer. |
 | Handler Latency (ms) | Incoming queue: The number of microseconds the incoming RPC requests spend in the worker queue before the beginning of processing.<br>Outbound queue time: The number of microseconds between an outbound call being created and put in the queue.<br>Outbound transfer time: The number of microseconds the outgoing traffic takes to exit the queue.<br>Note that Handler is a software implementation of a ring queue. | If this metric spikes or remains at a high level, it indicates a network issue or that the queues are full. |
```
Comment (krishkar99) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:102]: nit: To add YB-TServer to be consistent with the group ?

**Review by krishkar99 — CHANGES_REQUESTED** [review]

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 3]
```diff
@@ -61,30 +61,30 @@ The **Performance** tab provides the following metrics.
...
 
 ### Node / Resources
 
 | Graph | Description | Use |
 | :---| :--- | :--- |
 | CPU Usage (%) | The percentage of CPU use being consumed by the tablet or master server Yugabyte processes, as well as other processes, if any. In general, CPU usage is a measure of all processes running on the server. | High CPU use could indicate a problem and may require debugging by Yugabyte Support. |
-| Disk Usage (%) | Shows the amount of disk space provisioned for and used by the cluster. | Typically you would scale up at 80%, but consider this metric in the context of your environment. For example, usage can be higher on larger disks; some file systems issue an alert at 75% usage due to performance degradation. |
+| Disk Usage (GB) | Shows the amount of disk space provisioned for and used by the cluster. | Typically you would scale up at 80%, but consider this metric in the context of your environment. For example, usage can be higher on larger disks; some file systems issue an alert at 75% usage due to performance degradation. |
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:72]: I will link metrics with corresponding alerts on Alerts page to Alerts topic, and remove other references

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md (hunk 2 above)
Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:102]: Agree - but the UI has "Handler Latency", need to change it in the UI...

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 4]
```diff
@@ -93,97 +93,67 @@ The **Performance** tab provides the following metrics.
...
+| YB-TServer Change Config Latency (ms) | Latency of consensus change configuration processes. | You may consider this information while examining other metrics. |
+| YB-TServer Change Config RPCs | This metric is related to the RAFT Consensus Process. The number of times a peer was added or removed from the consensus group. | An increase in Change Config typically happens when YugabyteDB needs to move data around. This may happen as a result of a planned server addition or decommission, or a server crash looping. |
+| YB-TServer Context Switches | Voluntary context switches are writer processes that take a lock.<br>Involuntary context switches happen when a writer process has waited longer than a set threshold, which results in other waiting processes taking over. | A large number of involuntary context switches indicates a CPU-bound workload. |
+| YB-TServer Leader Elections Lost RPCs | This metric is related to the RAFT Consensus Process. The number of times a leader election has failed. | Leader election failures could be an indicator of a high CPU, blocked RPC queues, server restarts, and so on. <!--You should issue an alert on Leader Election Lost.--> |
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:113]: RAFT Consensus Process > Raft consensus process

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 5]
```diff
@@ -93,97 +93,67 @@ The **Performance** tab provides the following metrics.
...
+| YB-TServer Leader Step Down RPCs | This metric is related to the RAFT Consensus Process. The number of leader changes. | A Leader Step Down can indicate a normal change in leader, or it could be an indicator of a high CPU, blocked RPC queues, server restarts, and so on. <!--You should issue an alert on LeaderStepDown as a proxy for other system issues. --> |
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:114]: RAFT Consensus Process > Raft consensus process

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 6]
```diff
@@ -93,97 +93,67 @@ The **Performance** tab provides the following metrics.
...
+| YB-TServer WAL Operations/Sec | Group (Log Group Commit Count): the number of commits of an entire group, per second, per node.<br>Append (Log Append Count): the number of appends to the log segment file, per second, per node.<br>Sync (Log Sync Count): the number of syncs for the log segment file, per second, per node. | |
+| YB-TServer Consensus RPC Latency (ms) | Request Consensus Vote: latency of consensus request operations.<br>Update Consensus: latency of consensus update operations.<br>Multi-Raft Update Consensus: information pending. | If the value is high, it is likely that the overall latency is high.<br>This metric should be treated as a starting point in debugging the YB-Master and YB-TServer processes. |
+| YB-TServer Remote Bootstrap RPCs | The total count of remote bootstraps. | When a RAFT peer fails, YugabyteDB executes an automatic remote bootstrap to create a new peer from the remaining ones.<br>Bootstrapping can also be a result of planned user activity when adding or decommissioning nodes. <!--It is recommended to issue an alert on a change in this count outside of planned activity.--> |
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:118]: RAFT  >  Raft

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 7]
```diff
@@ -93,97 +93,67 @@ The **Performance** tab provides the following metrics.
...
+| YB-TServer Remote Bootstrap Connections | The total count of remote bootstrap connections. | When a RAFT peer fails, YugabyteDB executes an automatic remote bootstrap to create a new peer from the remaining ones.<br>Bootstrapping can also be a result of planned user activity when adding or decommissioning nodes. <!--It is recommended to issue an alert on a change in this count outside of planned activity.--> |
+| YB-TServer Leader Election RPCs | This metric is related to the RAFT Consensus Process. The number of times a leader has been elected. | |
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:120]: RAFT Consensus Process > Raft consensus process

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 8]
```diff
@@ -93,97 +93,67 @@ The **Performance** tab provides the following metrics.
...
+| YB-TServer Remote Bootstrap RPCs | The total count of remote bootstraps. | When a RAFT peer fails, YugabyteDB executes an automatic remote bootstrap to create a new peer from the remaining ones.<br>Bootstrapping can also be a result of planned user activity when adding or decommissioning nodes. <!--It is recommended to issue an alert on a change in this count outside of planned activity.--> |
+| YB-TServer Remote Bootstrap Connections | The total count of remote bootstrap connections. | When a RAFT peer fails, YugabyteDB executes an automatic remote bootstrap to create a new peer from the remaining ones.<br>Bootstrapping can also be a result of planned user activity when adding or decommissioning nodes. <!--It is recommended to issue an alert on a change in this count outside of planned activity.--> |
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:119]: RAFT  >  Raft

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 9]
```diff
@@ -93,97 +93,67 @@ The **Performance** tab provides the following metrics.
...
+| YB-TServer Leader Election RPCs | This metric is related to the RAFT Consensus Process. The number of times a leader has been elected. | |
+| YB-TServer Spinlock Contention Time (s) | Spinlock is a measurement of processes waiting for a server resource and using a CPU to check and wait repeatedly until the resource is available. | This value can become very high on large computers with many cores.<br>The GFlag `tserver_tcmalloc_max_total_thread_cache_bytes` is by default 256 MB, and this is typically sufficient for 16-core computers with less than 32 GB of memory. For larger computers, it is recommended to increase this to 1 GB or 2 GB.<br>You should monitor memory usage, as this requires more memory. |
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:121]: GFlag  >  I think we agree to use g-flag. Or maybe gflag. I recall talking about it.

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 10]
```diff
@@ -93,97 +93,67 @@ The **Performance** tab provides the following metrics.
...
-
-| Graph | Description | Use |
-| :---| :--- | :--- |
-| YSQL Operations/sec  | The number of various transaction control operations, such as BEGIN, COMMIT, ROLLBACK, as well as other operations through the YSQL API. | You may consider this information while examining other metrics. |
-| YSQL Average Latency (ms) | The average time taken by various transaction control operations, such as BEGIN, COMMIT, ROLLBACK, as well as other operations through the YSQL API. | You may consider this information while examining other metrics. |
--->
+### YB-Master server
 
-### Master server
+The [YugabyteDB Master Server](../../../architecture/concepts/yb-master/) (YB-Master) hosts system metadata, and records such as what tables exist in the system, where their tablets live, what users and roles exist, the permissions associated with them, and so on. YB-Masters are also responsible for coordinating background operations.
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:134]: The [YugabyteDB Master Server](../../../architecture/concepts/yb-master/) (YB-Master) hosts system metadata, and records such as what tables exist in the system, where their tablets live, what users and roles exist, the permissions associated with them, and so on.
Suggestion:
The [YugabyteDB Master Server](../../../architecture/concepts/yb-master/) (YB-Master) hosts system metadata, records about tables in the system and locations of their tablets, users, roles, permissions, and so on.

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 11]
```diff
@@ -194,3 +164,52 @@ DocDB uses a highly customized version of [RocksDB](http://rocksdb.org/), a log-
 | Compaction (B)             | The number of bytes being read and written to do compaction. If not a lot of data is being deleted, these levels are similar. In some cases, you might see a large delete indicated by large reads but low writes afterwards (because a large percentage of data was removed in compaction). | |
 | Compaction time (ms)           | Time (in milliseconds) for the compaction processes to complete. | |
 | Compaction num files           | The average number of files in any single compaction.        | You may consider this information while examining other metrics. |-->
+
+### DocDB
+
+[DocDB](../../../architecture/layered-architecture/#docdb) is a distributed document store, and the YugabyteDB storage layer. It uses a highly customized version of [RocksDB](http://rocksdb.org/), a log-structured merge tree (LSM)-based key-value store.
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:170]: Suggestion: 

[DocDB](../../../architecture/layered-architecture/#docdb) is a distributed document store that serves as the YugabyteDB storage layer. It uses a highly-customized version of [RocksDB](http://rocksdb.org/), a log-structured merge tree-based key-value store.

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 12]
```diff
@@ -194,3 +164,52 @@ DocDB uses a highly customized version of [RocksDB](http://rocksdb.org/), a log-
...
+| YB-TServer RocksDB Transaction Pool Cache | Percentage of transaction pool requests fulfilled by the transaction pool cache. | |
+| YB-TServer RocksDB Tablet Splitting Operations | Tracks tablet splits that have passed each of the following states:<br/>
+Add: Successfully submitted split RPC requests (applied on leader only)<br/>
+Apply: Successfully applied Raft split operations.<br/>
+Post Split Compactions: Number of successfully started post-split compactions. | If these metrics are high, a lot of tablet splits are being performed. These numbers should be roughly equal, and any large gap between them is anomalous. |
+| YB-TServer RocksDB Tablet Splitting Manager Runtime (ms) | How long the last automatic split manager run took. | The automatic split manager runs periodically to determine whether any new tablets are eligible for splitting. |
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:204]: How long the last automatic split manager run took.  >  The duration of time the last automatic split manager run took.

Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:204]: The automatic split manager runs periodically to determine whether any new tablets are eligible for splitting.  >  The automatic split manager runs periodically to determine whether or not any new tablets are eligible for splitting.

**Review by lizayugabyte — COMMENTED** [review]
Some comments

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md [hunk 13]
```diff
@@ -93,97 +93,67 @@ The **Performance** tab provides the following metrics.
 | Transaction | The number of transactions. | This value depends on the application or activity. Because transactions can have batched statements, no specific guidance is possible for this metric. |
 | Inbound RPC Connections Alive | The count of current connections at the CQL API level. | If this spikes to a number much higher than average, you should consider that there may be an active DDoS or a security incident. |-->
 
-### Tablet server
+### YB-Tablet server
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:96]: Like this in the UI - which should probably be changed

File: docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md:121]: (s) is the units - seconds

[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md]: The are "sec" in all other places in the same file, I think

[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md]: I mean it changes depending on the metric in the UI. Docs reflect the metric name as displayed. 's' appears to be the actual standard going from wikipedia.

**Pull request merged**